src/components/inputs/Search/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as SearchIcon } from '../../../icons/Search.svg';
+import './styles.css';
+import { searchBar } from '../../../../types';
+
+function SearchBar({ searchButtonAction }: searchBar) {
```

Comment:
let's also allow `headerText` and `footerText` to be optional props to pass in the text fields from the figma. for readability, I think it helps removing the destructring of props down into the function.

e.g.
```
function SearchBar(props: SearchBarProps){
    const { searchBarAction, headerText, footerText } = props;
    ...
}
```